src/character.cpp

```diff
@@ -1289,6 +1289,23 @@ std::array<encumbrance_data, num_bp> Character::calc_encumbrance( const item &ne
     return ret;
 }
 
+int Character::get_weight() const
+{
+    int ret = 0;
+    int wornWeight;
+    wornWeight = std::accumulate(worn.begin(),
+                                 worn.end(),
+                                 0,
+                                 [](int sum, const item& itm) { return sum + itm.weight(); }
+                                 );
```

Comment:
This newline spam looks bad, 3 first arguments should be on one line, with 4th on 3 next lines.